ci/build_container/build_container.sh

```diff
@@ -19,6 +19,17 @@ apt-get update
 apt-get install -y bazel
 rm -rf /var/lib/apt/lists/*
 
+# Build a version of Bazel suitable for C++ code coverage collection. See
+# https://github.com/bazelbuild/bazel/issues/1118 for why we need this. This is the envoy-coverage
+# branch on the cloned repository.
+git clone https://github.com/htuch/bazel.git /tmp/bazel-coverage
```

Comment:
I've requested guidance on the clean way to get this upstream in bazelbuild/bazel#1118.